system/Debug/Exceptions.php

```diff
@@ -78,6 +90,9 @@ public function __construct(\Config\Exceptions $config)
 		$this->viewPath = rtrim($config->errorViewPath, '/ ') . '/';
 
 		$this->config = $config;
+
+		$this->request  = Services::request();
```

Comment:
Would prefer these to be passed in as method parameters. Please modify `system/Config/Services` to pass it in.